test/unit/stubs/style.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
What's the reason for including this? Please add the explanation as a comment in this file :)

Reply:
resolved

Reply:
What's the resolution here? I don't see a comment explaining what this file does?

Reply:
Removed added resolution in jest-config